packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -267,6 +287,33 @@ describe('Sayt Component', () => {
     });
   });
 
+  describe('getDebounce()', () => {
+    it('should return a debounced callback method', () => {
+      const delay = sayt.debounce;
```

Comment:
I think it would be better to set this to a known concrete value. For example, you can set it to 200.